ci/README.md

```diff
@@ -48,6 +48,8 @@ As of November 2019 after [#8859](https://github.com/envoyproxy/envoy/pull/8859)
 To override the C++ standard library in your build, set environment variable `ENVOY_STDLIB` to `libstdc++` or `libc++` and
 run `./ci/do_ci.sh` as described below.
 
+Support Clang+LLVM package from [LLVM official site](http://releases.llvm.org/download.html) from 11.0.1 or higher.
```

Comment:
I concur with Lizan, if you look two sections higher, under "Build image base and compiler versions" we explain this. It doesn't need to be called out under thit C++ standard library section. (That section too is out of date.) I think the change is superfluous, there is already too much overlapping documentation of the recommended tools.

See https://www.envoyproxy.io/docs/envoy/latest/start/building#linux-mac-target-requirements for the official location of such details.

Reply:
Oh, seems we'd better remove redundant contents? @wrowe

Reply:
Yes, I agree we should streamline the prerequisites for the published docs and internally within the source tree. That seems like a topic for a distinct PR, help is welcome!